subsys/bluetooth/services/nus.c

```diff
@@ -15,6 +15,22 @@ LOG_MODULE_REGISTER(bt_nus, CONFIG_BT_NUS_LOG_LEVEL);
 
 static struct bt_nus_cb nus_cb;
 
+static void nus_ccc_cfg_changed(const struct bt_gatt_attr *attr,
+				  uint16_t value)
+{
+	if (value == BT_GATT_CCC_NOTIFY) {
```

Comment:
something like this would use less rom :
```
if (nus_cb.notification_enabled) {
       LOG_DBG("Notification has been turned %s", value == BT_GATT_CCC_NOTIFY ? "on" : "off");
	nus_cb.notification_enabled(value == BT_GATT_CCC_NOTIFY ?
              BT_NUS_NOTIFY_STATUS_CCCD_ENABLED : BT_NUS_NOTIFY_STATUS_CCCD_DISABLED) ;
}

Reply:
Done